projects/dotcms-field-elements/src/components/dot-multi-select/dot-multi-select.e2e.ts

```diff
@@ -1,100 +1,337 @@
+                element.setProperty('label', value);
+                await page.waitForChanges();
+
+                const labelElement = await dotTestUtil.getDotLabel(page);
+                expect(await labelElement.getProperty('label')).toBe(value);
+            });
+
+            it('should not break with invalid data', async () => {
+                const wrongValue = [1, 2, '3'];
+                element.setProperty('label', wrongValue);
+                await page.waitForChanges();
+
+                const labelElement = await dotTestUtil.getDotLabel(page);
+                expect(await labelElement.getProperty('label')).toEqual([1, 2, '3']);
+            });
+        });
+
+        describe('hint', () => {
+            it('should render hint', async () => {
+                const value = 'test';
+                element.setProperty('hint', value);
+                await page.waitForChanges();
+
+                const hintElement = await dotTestUtil.getHint(page);
+                expect(hintElement.innerHTML).toBe(value);
+            });
+
+            it('should not render hint', async () => {
+                const hintElement = await dotTestUtil.getHint(page);
+                expect(hintElement).toBeNull();
+            });
+
+            it('should not break and not render with invalid data', async () => {
+                const wrongValue = [{ a: 1 }];
+                element.setProperty('hint', wrongValue);
+                await page.waitForChanges();
+
+                const hintElement = await dotTestUtil.getHint(page);
+                expect(hintElement).toBeFalsy();
+            });
+        });
+
+        describe('options', () => {
+            it('should render options', async () => {
+                const value = 'a|1,b|2,c|3';
+                element.setProperty('options', value);
+                await page.waitForChanges();
+
+                const optionElements = await getOptions(page);
+                expect(optionElements.length).toBe(3);
+            });
+
+            it('should not render options', async () => {
+                const optionElements = await getOptions(page);
+                expect(optionElements).toEqual([]);
+            });
+
+            it('should not break with invalid data', async () => {
+                const wrongValue = [{ a: 1 }];
+                element.setProperty('options', wrongValue);
+                await page.waitForChanges();
+
+                const optionElements = await getOptions(page);
+                expect(optionElements).toEqual([]);
+            });
+        });
+
+        describe('required & requiredMessage', () => {
+            it('should render required attribute in label and dot-required css class', async () => {
+                element.setProperty('required', true);
+                await page.waitForChanges();
+                const labelElement = await dotTestUtil.getDotLabel(page);
+                expect(element).toHaveClasses(['dot-required']);
```

Comment:
We should not care if the `dot-label` have a class, we only care that we're passing the right `required` attr.